src/components/App/App.js

```diff
@@ -49,12 +53,15 @@ function App() {
             src={pcic_logo}
             width='328'
             height='38'
-            alt='Pacific Climate Impacts Consotrium'
+            alt='Pacific Climate Impacts Consortium'
           />
         </a>
         <div className="Title">
           <h1>Salmon Climate Impacts Portal</h1>
         </div>
+        <div className="Help">
+            <a href={`${process.env.PUBLIC_URL}/general_help.pdf`}>General Help</a>
```

Comment:
This file appears to be missing from `./public/` and the link currently does nothing. 

Also maybe add `target="_blank"` attribute so that it opens in a new tab?